src/installation/guides/fde.md

```diff
@@ -45,9 +45,10 @@ Device      Start       End   Sectors  Size Type
 
 ## Encrypted volume configuration
 
-[Cryptsetup](https://man.voidlinux.org/cryptsetup.8) defaults to LUKS2, yet GRUB
-releases before 2.06 only had support for LUKS1. Therefore, it might make sense
-to force LUKS1 if you wish to achieve better compatibility.
+GRUB 2.06 added partial support for LUKS2, however LUKS2 will not work with GRUB
+by default. See the [Arch Wiki](https://wiki.archlinux.org/title/GRUB#LUKS2) for
```

Comment:
We should mention that only PBKDF2 is supported and it won't work with the default Argon2i instead of linking to a whole wiki page. It's not that complicated.